lsp-completion.el

```diff
@@ -715,6 +706,12 @@ The CLEANUP-FN will be called to cleanup."
   "Disable LSP completion support."
   (lsp-completion-mode -1))
 
+(defun lsp-completion-pass-through-try-completion (string table pred _point)
```

Comment:
Why not call it `lsp-completion-passthrough-try-completion`? "passthrough" is a word - why add the hypen?